example/lib/main.dart

```diff
@@ -94,9 +100,22 @@ class _MyHomePageState extends State<MyHomePage> {
           endHour: 23,
           use24HourFormat: false,
           style: TimePlannerStyle(
-            // cellHeight: 60,
-            // cellWidth: 60,
-            showScrollBar: true,
+            cellHeight: 120,
+            cellWidth: 282,
+              showScrollBar: false,
+
+            dividerColor: Colors.transparent,
+            headerContainerDecoration: BoxDecoration(
+              color: Colors.white,
+              boxShadow: [
+                BoxShadow(
+                  color: const Color(0xff030303).withOpacity(0.08),
+                  blurRadius: 12,
+                  spreadRadius: 0,
+                  offset: Offset(0, 4),
+                )
+              ]
+            )
           ),
           headers: const [
             TimePlannerTitle(
```

Comment:
Can we center the header vertically? 

<img width="283" alt="Screen Shot 2023-02-28 at 6 05 24 PM" src="https://user-images.githubusercontent.com/84877782/222025959-fc19799b-857d-422a-9665-f5f506e23050.png">

Reply:
Also are we using General Sans? The font looks different in Figma https://www.figma.com/file/utRy1gg3Zv9X4hFnQL8Uzz/Frontstage-App-UX%2FUI?node-id=729%3A8300&t=pVl301Hauxz0yr1f-1